roles/neutron-data-network/tasks/monitoring.yml

```diff
@@ -17,11 +17,19 @@
 - name: ipchanged process check
   sensu_process_check: service=/usr/local/sbin/ipchanged
 
-- name: ucarp failover alert
+- name: ucarp failover alert - ubuntu
   sensu_check: name=check-ucarp-failover plugin=check-log.rb use_sudo=true
                auto_resolve=false interval=20 occurrences=1
                args="-f /var/log/syslog -q 'ucarp.+[Ss]witching to state' --silent"
   notify: restart sensu-client
+  when: os == 'ubuntu'
+
+- name: ucarp failover alert - rhel
```

Comment:
looks like only difference is log file name.. we should avoid duplicate code and instead add logic to smartly inject right logfile name...

"{{ ( os == 'ubuntu' )| ternary('/var/log/syslog', '/var/log/messages') }}"